### PR TITLE
Tag Page Metatags

### PR DESCRIPTION
**Pull request by Drapegnik**
http://metatags.io

Screen Shot 2020-01-22 at 12 29 45 AM
Screen Shot 2020-01-22 at 12 29 54 AM
Screen Shot 2020-01-22 at 12 31 28 AM
Screen Shot 2020-01-22 at 12 30 38 AM

### CODE REVIEW

**Comment by Drapegnik** [comment]
[to fix Tag Page title](https://trello.com/c/ktMZ3HF7/443-to-fix-tag-page-title)

File: components/social/Metatags.js [hunk 1]
```diff
@@ -35,20 +35,25 @@ export const MetaDescription = ({ description }) => (
   </Head>
 );
 
-export const MetaImage = ({ url }) => (
+export const DEFAULT_IMAGE =
+  'https://res.cloudinary.com/wir-by/image/upload/v1542061416/production/meta/meta-wir.jpg';
```
Comment (UladBohdan) [inline components/social/Metatags.js:39]: I've reminded Vitalik we need a new one.

Comment (UladBohdan) [reply components/social/Metatags.js:39]: ... and we can expect an update tomorrow night.

**Comment by Drapegnik** [comment]
[compact article / tag page / topic page preview for (at least) telegram instead of large-picture one](https://trello.com/c/Rgu2A2XT/416-compact-article-tag-page-topic-page-preview-for-at-least-telegram-instead-of-large-picture-one)

**Pull request merged**